source/puppet/5.3/hiera_automatic.md

```diff
@@ -135,71 +140,82 @@ puppet lookup [--help] [--type <TYPESTRING>] [--merge first|unique|hash|deep] [-
 
 ### Examples
 
-To look up 'key_name' using the Puppet Server node's facts: 
+To look up 'key_name' using the Puppet Server node's facts:
 
-`$ puppet lookup key_name`
```

Comment:
I didn't remove the prompt (`$ `) here, but should have. Adding prompts to commands makes them more difficult for users to copy and paste them if necessary.